babel.config.js

```diff
@@ -1,13 +1,11 @@
 module.exports = api => {
-    const config = { presets: ['@magento/peregrine'] };
-    if (process.env.CODEBUILD_BUILD_ID) {
-        // In AWS CodeBuild CI, don't cache
-        api.cache.never();
-    } else if (api.env() === 'development') {
+    const config = {
+        presets: ['@magento/peregrine'],
+        exclude: [/packages\/babel\-preset\-peregrine\//]
```

Comment:
This is odd -- do we do this because we co-locate the preset with other packages that use the preset? Why not just directly reference the preset by `presets: ['packages/babel-preset-peregrine']` and just have the `create-pwa-app` command use the `@magento/peregrine` tag?

Reply:
@sirugh I hadn't thought of that! If you wanna try it, please be my guest. Otherwise, we can call that a later refactor. It'll simplify the build and maybe even speed it up.

Reply:
We can do it later :P